library/src/main/java/bio/terra/landingzone/library/landingzones/definition/factories/ParametersResolver.java

```diff
@@ -1,28 +1,10 @@
 package bio.terra.landingzone.library.landingzones.definition.factories;
 
-import java.util.HashMap;
 import java.util.Map;
 
-public class ParametersResolver {
-  private final Map<String, String> defaultParameters;
-  private final Map<String, String> parameters;
-
-  public ParametersResolver(Map<String, String> parameters, Map<String, String> defaultParameters) {
-    if (defaultParameters == null) {
-      this.defaultParameters = new HashMap<>();
-    } else {
-      this.defaultParameters = defaultParameters;
-    }
-
-    this.parameters = parameters;
-  }
+public record ParametersResolver(Map<String, String> parameters) {
```

Comment:
I question the need for this class going forward if it's a dumb wrapper around Map<String, String>

Reply:
Yeah, I toyed with removing it. I only kept it in as keeping `getValue` with its `""` default made the required changes smaller